schema.sql

```diff
@@ -1,5 +1,23 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
+     animal_name TEXT,
+     DATE_OF_BIRTH DATE,
+     ESCAPE_ATTEMPTS INT,
+     NEUTERED BIT,
+     WEIGHT_KG DECIMAL,
+     PRIMARY KEY (id)
 );
+
+ALTER TABLE animals ADD species varchar(250);
+
```

Comment:
- [ ] Great job so far 👏🏻  You should create a many-to-many relationship between the tables species and vets: a vet can specialize in multiple species, and a species can have multiple vets specialized in it. Kindly make you you create a "join table" called specializations to handle this relationship.
-  eg:
   ```sql
      CREATE TABLE specializations (
      species_id int,
      vet_id int
      );
   ```